### PR TITLE
Descriptive names on accordions

### PR DESCRIPTION
**Pull request by RDIL**
* MUI v5 changes `ExpansionPanel` -> `Accordion`, update as such
* Add names to the different actions (todo: possibly icons in the future?)

Fixes https://github.com/cirruslabs/cirrus-ci-docs/issues/676

Signed-off-by: Reece Dunham <me@rdil.rocks>

### CODE REVIEW

**Comment by RDIL** [comment]
cc @alexwayfer @fkorotkov

File: src/components/tasks/TaskCommandList.tsx [hunk 1]
```diff
@@ -58,16 +58,36 @@ class TaskCommandList extends React.Component<Props> {
...
+      topText = <Typography variant="body1">Run {command.name}</Typography>;
+    } else if (command.type === 'CACHE') {
+      topText = <Typography variant="body1">Populate {command.name} cache</Typography>;
+    } else if (command.type === 'UPLOAD_CACHE') {
+      // the upload text is added on the backend
+      topText = <Typography variant="body1">{command.name}</Typography>;
+    } else if (command.type === null) {
+      // todo: remind Fedor to fix this on the backend
+      topText = <Typography variant="body1">Upload {command.name} artifacts</Typography>;
+    } else {
```
Comment (AlexWayfer) [inline src/components/tasks/TaskCommandList.tsx:77]: I see the same `<Typography variant="body1">` in each case, so maybe we should take it out? And assign only different parts to `topText`. And wrap the `topText` usage into `<Typography variant="body1">`.

File: src/components/tasks/TaskCommandList.tsx [hunk 2]
```diff
@@ -58,16 +58,36 @@ class TaskCommandList extends React.Component<Props> {
...
+      throw new Error(`no way to handle ${command.type}!`);
+    }
```
Comment (AlexWayfer) [inline src/components/tasks/TaskCommandList.tsx:79]: Also, about "case": maybe here [`switch`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) would be better?

File: src/components/tasks/TaskCommandList.tsx [hunk 3]
```diff
@@ -58,16 +58,36 @@ class TaskCommandList extends React.Component<Props> {
...
+    if (command.type === 'CLONE') {
+      topText = 'Clone';
+    } else if (['EXECUTE_SCRIPT', 'EXECUTE_BACKGROUND_SCRIPT'].includes(command.type)) {
+      topText = 'Run ' + command.name;
+    } else if (command.type === 'CACHE') {
+      topText = 'Populate ' + command.name + ' cache';
+    } else if (command.type === 'UPLOAD_CACHE') {
+      // the upload text is added on the backend
+      topText = command.name;
+    } else if (command.type === null) {
```
[the remark this answers is not in the document]
Comment (RDIL) [reply src/components/tasks/TaskCommandList.tsx:74]: @fkorotkov I can’t, the Cirrus CI server returns `null` as the `TaskCommandType` for artifacts, see https://github.com/cirruslabs/cirrus-ci-web/blob/dd36c2e3972b66126ccd13190b4ac1e6f4545582/schema.graphql#L656

[the remark this answers is not in the document]
Comment (fkorotkov) [reply src/components/tasks/TaskCommandList.tsx:74]: I'll fix it on the backend and will push a fix to your branch

[the remark this answers is not in the document]
Comment (fkorotkov) [reply src/components/tasks/TaskCommandList.tsx:74]: @RDIL don't have permissions to push but now ARTIFACTS is available through the API

[the remark this answers is not in the document]
Comment (RDIL) [reply src/components/tasks/TaskCommandList.tsx]: Fixed

**Pull request merged**